.gitlab-ci.yml

```diff
@@ -52,6 +52,7 @@ build_preview:
     #- remove_static_from_repo
     #- create_artifact
     #- create_artifact_untracked
+    #- ./local/bin/py/missing_metrics.py -k $(get_secret 'dd-demo-api-key') -p $(get_secret 'dd-demo-app-key') $(get_secret 'dd-corp-api-key') $(get_secret 'dd-corp-app-key')
```

Comment:
So then i think this would update to be like this. Only thing is i'm not sure we have dd_app_key setup i'll have to try look into that

```
./local/bin/py/missing_metrics.py -k $(get_secret 'dd-demo-api-key') -p $(get_secret 'dd-demo-app-key') -a $(get_secret 'dd_api_key') -b $(get_secret 'dd_app_key')
```

Reply:
also a bit further up we have

```
#- sync_integration_descriptions
- sync_integration_descriptions_cached
```

you might want to switch the comment the other way around so cached is off and the top one is on. This makes it so it pulls all the files from github on each preview site build instead of using an artifact. That way all the csv will be there in /tmp